documentation/asciidoc/computers/camera/external_trigger.adoc

```diff
@@ -0,0 +1,66 @@
+== External Trigger on the Global Shutter Camera
+The global shutter camera can be triggered externally by pulsing the external trigger (denoted on the board as XTR) connection on the board.
+Multiple cameras can be connected to the same pulse, allowing for an alternative way to synchronise two cameras.
+
+The exposure time is equal to the low pulse-width time plus an additional 14.26us. i.e. a low pulse of 10000us leads to an exposure time of 10014.26us.
+Framerate is directly controlled by how often you pulse the pin. A PWM frequency of 30Hz will lead to a framerate of 30 frames per second.
+
+In this example, we will be using a Raspberry Pi Pico to create the pulses that will drive the camera modules.
+
+image::images/external_trigger.jpg[Image showing pulse format]
+
+=== Preparation
+
+NOTE: Soldering wires to your camera module will void the camera's warranty. When soldering to the Global Shutter Camera, please remove the plastic back cover to avoid damaging the cover.
+
+First, remove/desolder resistor 11 on the GS Camera board. This connects GP0 to XTR and without removal, the camera will not operate
+in external trigger mode.
+The location of the resistor is displayed below.
+
```

Comment:
Just one small change needed here:

~First, remove/desolder resistor 11 on the GS Camera board. This connects GP0 to XTR and without removal, the camera will not operate in external trigger mode.~ If you have transistor Q2 fitted, remove/desolder the component along with R11. This is needed as the drive strength on the Raspberry Pi Pico is not strong enough to bring the XTRIG line low, and the the camera will not operate in external trigger mode.

We also want to circle the Q2 in the image below.